main/settings.py

```diff
@@ -981,6 +981,24 @@
     description="CyberSource access key",
 )
 
+MITOL_PAYMENT_GATEWAY_CYBERSOURCE_MERCHANT_ID = get_string(
```

Comment:
i think we should actually upstream these settings into `mitol-django-payment-gateway` as a boilerplate settings file. You can take a look at https://github.com/mitodl/ol-django/blob/main/src/mitol/google_sheets/settings/google_sheets.py as an example. Can you move the other `MITOL_PAYMENT_GATEWAY_` ones in addition to the ones you're adding?

Reply:
As per [my comment here](https://github.com/mitodl/mitxonline/pull/599#issuecomment-1155166560), These settings have already been moved to ol-django in [this PR](https://github.com/mitodl/ol-django/pull/64/files#diff-8925307c256c81211d35d1a950207fb606d513e45d72cc6ab6ccaa4c44f5d431R1). Since we haven't yet released that I'm kept these here for testing because the ol-django packages can't directly be installed right from git because of pants build and without these there are settings errors.

These should be removed once that ol-django PR is merged and released.

The priority from my side would be to get the ol-django PR reviewed/merged/deployed first.